README.md

```diff
@@ -65,24 +66,45 @@ Create a file named `kroki.html` with the following content and open it in your
       var kroki = AsciidoctorKroki
 
       const registry = asciidoctor.Extensions.create()
-      kroki.register(registry)
+      kroki.register(registry) // <1>
       var result = asciidoctor.convert(input, {'extension_registry': registry})
       document.getElementById('content').innerHTML = result
     </script>
   </body>
 </html>
 ```
-**<1>** Register the extension in the global registry <br/>
-**<2>** Register the extension in a dedicated registry
+**<1>** Register the extension in a dedicated registry
 
 ## Usage
 
-```adoc
-[plantuml,alice-bob,svg,role=sequence]
-....
-alice -> bob
-....
+Kroki currently supports the following diagram libraries:
+
+* ActDiag: `actdiag`
+* BlockDiag: `blockdiag`
+* BPMN: `bpmn`
+* Bytefield: `bytefield`
+* C4 (PlantUML): `c4plantuml`
+* Ditaa: `ditaa`
+* ERD: `erd`
+* GraphViz : `graphviz`
+* Mermaid: `mermaid`
+* Nomnoml: `nomnoml`
+* NwDiag: `nwdiag`
+* PacketDiag: `packetdiag`
+* PlantUML: `plantuml`
+* RackDiag: `rackdiag`
+* SeqDiag: `seqdiag`
+* SVGBob: `svgbob`
+* UMLet: `umlet`
+* Vega: `vega`
+* Vega-Lite: `vegalite`
+* WaveDrom: `wavedrom`
+
+In your AsciiDoc document, you can either declare your diagram inline or reference a diagram file using the corresponding macro or using the `include` directive.
```

Comment:
```suggestion
In your AsciiDoc document, you can either write your diagram inline or alternatively you can make a reference to the diagram file using macro form or with the `include` directive.
```
Too many "ors" running together, just a suggestion...

Reply:
Thanks that's better!